HACKING.txt

```diff
@@ -51,58 +66,43 @@ Coding Style
 - Pyramid uses Black (https://pypi.org/project/black/) and isort (https://pypi.org/project/isort/) for code formatting style.
   To run formatters:
 
-    $ tox -e format
+    $ $TOX -e format
```

Comment:
I prefer not to use environment variables.

Reply:
`$VENV` used to exist before. It's the easiest way I can make sure that users that are copy/pasting commands will get the right `tox`.

Reply:
For total newbs in narrative and tutorial docs, we definitely need `$VENV` with all of its context about a virtual environment. For the audience of HACKING.txt, is `$VENV` the best way? I dunno.

Perhaps a disclaimer like, "For brevity with commands, we assume you know where your `tox` is installed and how to invoke it" placed before the first instance of invoking tox?